styles.css

```diff
@@ -0,0 +1,110 @@
+:root {
+  --gris: #252525;
+  --blanco: #fff;
+  --naranja: #d4440b;
+  --madera: #eed09d;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-size: 62.5%;
+}
+
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: var(--madera);
+}
+
+.magazine-container {
+  margin-top: 1%;
+  width: 620px;
+  background: var(--gris);
+  height: 670px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.cover-container {
+  background: url(https://images.unsplash.com/photo-1572304290488-9ea38ef644a9?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=634&q=80)
+    no-repeat center center;
+  background-size: cover;
+  width: 600px;
+  background-size: 100% 100%;
+  height: 650px;
+  display: grid;
+  grid-template-columns: 25% auto 25%;
+  grid-template-rows: 25% 20% 45% 5%;
+}
+
+.title-container {
+  grid-column: 1 / 5;
+  grid-row: 1/1;
+  display: flex;
+  align-items: stretch;
+  justify-content: center;
+}
+
+.title-container .title {
+  color: var(--gris);
+  font-size: 7rem;
```

Comment:
great !!
siempre es buena idea manejar los font size con REM 
more info aca ---> 

https://stackoverflow.com/questions/30374863/why-use-rem-instead-px-when-its-the-same-anyway
![image](https://user-images.githubusercontent.com/81988661/121558251-1c045980-c9db-11eb-9df9-717d0664ecb2.png)